tests/attention/test_cute_dsl_mla_decode.py

```
@@ -852,6 +873,105 @@ def test_cute_dsl_mla_decode_attention_sink(batch_size, seq_len_k, page_size):
+        query,
+        kv_cache,
+        block_tables,
+        seq_lens,
+        workspace_buffer,
+        num_heads,
+        latent_dim,
+        rope_dim,
+    ) = _make_mla_test_data(batch_size=1, seq_len_k=128, page_size=64, dtype=dtype)
+    sink = torch.randn((num_heads,), dtype=dtype, device="cuda")
+
+    with pytest.raises(ValueError, match="monolithic.*sinks.*modular"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)